src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -16,8 +16,8 @@ public class When_broadcasting_a_command : NServiceBusAcceptanceTest
         {
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Sender>(b => b.When(c => c.EndpointsStarted, (session, c) => session.Send(new Request())))
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("1")); })
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("2")); })
+                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.AddUniquelyAddressableQueue("1")); })
```

Comment:
Thinking about it a bit more, I suppose the counter argument could be, let's go with `MakeInstanceUniquelyAddressable` because that is high-level view of what that API is doing, without going into the specifics.

Then, as it is currently in the PR, the intellisense xml comments could point out more about what the method does and how the `discriminator` parameter is used. This would let us indicate that the it's used to create an additional instance-specific queue.